src/calculateRank.js

```diff
@@ -67,23 +64,13 @@ function calculateRank({
 
   if (normalizedScore < RANK_S_VALUE) {
     level = "S+";
-  }
-  if (
-    normalizedScore >= RANK_S_VALUE &&
-    normalizedScore < RANK_DOUBLE_A_VALUE
-  ) {
+  } else if (normalizedScore < RANK_DOUBLE_A_VALUE) {
     level = "S";
-  }
-  if (
-    normalizedScore >= RANK_DOUBLE_A_VALUE &&
-    normalizedScore < RANK_A2_VALUE
-  ) {
+  } else if (normalizedScore < RANK_A2_VALUE) {
     level = "A++";
-  }
-  if (normalizedScore >= RANK_A2_VALUE && normalizedScore < RANK_A3_VALUE) {
+  } else if (normalizedScore < RANK_A3_VALUE) {
     level = "A+";
-  }
-  if (normalizedScore >= RANK_A3_VALUE && normalizedScore < RANK_B_VALUE) {
+  } else if (normalizedScore < RANK_B_VALUE) {
     level = "B+";
```

Comment:
Was the B+ intended to be the lowest rank available? I thought there used to be A

Reply:
Yes, the B+ is the lowest and the A is not available now. There was the rank A in the first version, which has been changed to B+.
(please check the [history for src/calculateRank.js](https://github.com/anuraghazra/github-readme-stats/commits/master/src/calculateRank.js))

This PR is simply a refactoring, so the ranking algorithm has not been changed.